java/test/jmri/jmrit/beantable/signalmast/AddSignalMastPanelTest.java

```diff
@@ -28,6 +32,7 @@ public void testDefaultSystems() {
 
     @Test
     public void testSearch() throws Exception {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         try {  // need try-finally to ensure junk deleted from user area
```

Comment:
I would also check and see if this test needs the headless check.

Also, since you have the try...finally clock, put in a catch that fails the test ( with an Assert.fail ) if an exception is thrown in the try block.

Reply:
Removed (we'll see if any integration checks fail)